scene/resources/particle_process_material.cpp

```diff
@@ -424,7 +434,8 @@ void ParticleProcessMaterial::_update_shader() {
 		code += "	float s2 = noise(p - vec4(vec3(0.0), 1.7320508 * 2048.333333));\n";
 		code += "	vec3 c = vec3(s, s1, s2);\n";
 		code += "	return c;\n";
-		code += "}\n";
+		code += "}\n\n";
+
```

Comment:
I aimed to standardize vertical spacing like we do in engine code, with an empty line between each function definition, or between separate logic blocks.

In the codegen, whenever we have a `\n\n`, it should be followed by an actual empty line to help make this obvious.